app/Composers/DashboardComposer.php

```diff
@@ -26,6 +26,42 @@
  */
 class DashboardComposer
 {
+    private $componentCount;
```

Comment:
These variables are missing docblocks. For consistency, we also use `protected`.